index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:wght@400;500&family=Poppins:wght@400;500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Personal Portfolio Website</title>
+</head>
+<body>
+    <div id="header">
+       <div class="mobile-header">
+            <div id="primary-logo">
+                <div id="mobile-logo">
+                    <img src="images/Shape1.png" id="shape1" alt="logo-shape1">
+                    <img src="images/Shape.png"  id= "shape" alt="logo-shape">
+                </div>
+                <div id="logo"></div>
+            </div>
+            <div id="mobile-menu">
+                <img src="images/ic_menu.png" id="menu-icon" alt="menu-icon" >
+            </div>
+        </div>
+        <div class="main-header">
+            <div class="main-menu">
+                <ul>
+                    <li><a href="#"> Home</a></li>
+                    <li><a href="#">Portfolio</a></li>
+                    <li><a href="#">About</a></li>
+                    <li><a href="#">Contact</a></li>
+                </ul>
+            </div>
+        </div>
+       </div>
+    </div>
+    <div class="headline">
+       <div>
+        <div id="primary-text">
+            <h2>Hi👋, I'm Cephas Tokoli</h2>
```

Comment:
> * [ ]  Kindly use the `h1` tag over the `h2` tag because it is the main heading of your portfolio website. Using the `<h1>` tag for the main heading can also improve the overall _user experience_ of the website. By clearly identifying the main topic or purpose of the page, users can quickly determine whether the content is relevant to their needs and interests.

Because this is the main title of the website, kindly use a h1 tag as requested by the previous reviewer.